api/src/controllers/CLE/referent.js

```diff
@@ -0,0 +1,77 @@
+const express = require("express");
+const passport = require("passport");
+const router = express.Router();
+const Joi = require("joi");
+const crypto = require("crypto");
+const { capture } = require("../../sentry");
+const { SUB_ROLES, ROLES, SENDINBLUE_TEMPLATES } = require("snu-lib");
+const { ERRORS, inSevenDays } = require("../../utils");
+const { sendTemplate } = require("../../sendinblue");
+const config = require("../../config");
+const EtablissementModel = require("../../models/ClasseEngagee/etablissement");
+const ReferentModel = require("../../models/referent");
+
+router.post("/invite-coordonnateur", passport.authenticate("referent", { session: false, failWithError: true }), async (req, res) => {
+  try {
+    const { error, value } = Joi.object({
+      email: Joi.string().lowercase().trim().email().required(),
+      firstName: Joi.string().required(),
+      lastName: Joi.string().required(),
+    })
+      .unknown()
+      .validate(req.body, { stripUnknown: true });
+
+    if (error) {
+      capture(error);
+      return res.status(400).send({ ok: false, code: ERRORS.INVALID_BODY });
+    }
+
+    const { email, firstName, lastName } = value;
+
+    //for now, only one chef per etablissement
+    const etablissement = await EtablissementModel.findOne({ chefIds: req.user._id });
+    if (!etablissement) {
+      return res.status(404).send({ ok: false, code: ERRORS.NOT_FOUND });
+    }
+
+    //for now, only one sous-chef per etablissement
+    if (etablissement.sousChefIds.length > 0) {
```

Comment:
Juste ça devrait suffire 

```suggestion
    if (etablissement.sousChefIds.length) {
```